internal/app/connectconformance/testsuites/data/basic.yaml

```diff
@@ -80,22 +69,20 @@ testCases:
         - name: x-custom-trailer
           value: ["bing"]
 - request:
-    testName: server-stream/no-response
-    streamType: STREAM_TYPE_SERVER_STREAM
-    requestMessages:
-    - "@type": type.googleapis.com/connectrpc.conformance.v1.ServerStreamRequest
-      responseDefinition:
-        responseHeaders:
-        - name: x-custom-header
-          value: ["foo"]
-        responseTrailers:
-        - name: x-custom-trailer
-          value: ["bing"]
-- request:
-    testName: server-stream/no-definition
```

Comment:
In this case, these two were similar (just like the two unary cases above that were removed). But I left one of them because they do check something interesting for server and bidi streams: response streams that have zero messages.